templates/ts-apollo-fullstack/server/src/schema/schema.graphql

```diff
@@ -36,25 +43,30 @@ type Note {
   comments: [Comment]!
 }
 
-input NoteInput {
+input NoteData {
+  id: ID
+  title: String!
+  description: String
+}
+
+input NoteFilter {
   id: ID
   title: String
   description: String
 }
 
 type Query {
-  test: String @deprecated
   findAllNotes(limit: Int, offset: Int): [Note]!
-  findNotes(fields: NoteInput, limit: Int, offset: Int): [Note]!
+  findNotes(filter: NoteFilter, limit: Int, offset: Int): [Note]!
   findAllComments(limit: Int, offset: Int): [Comment]!
-  findComments(fields: CommentInput, limit: Int, offset: Int): [Comment]!
+  findComments(filter: CommentFilter, limit: Int, offset: Int): [Comment]!
 }
 
 type Subscription {
-  newNote(input: NoteInput): Note!
-  updatedNote(input: NoteInput): Note!
-  deletedNote(input: NoteInput): Note!
-  newComment(input: CommentInput): Comment!
-  updatedComment(input: CommentInput): Comment!
-  deletedComment(input: CommentInput): Comment!
+  newNote(data: NoteData): Note!
+  updatedNote(data: NoteData): Note!
+  deletedNote(filter: NoteFilter): Note!
+  newComment(data: CommentData): Comment!
+  updatedComment(data: CommentData): Comment!
+  deletedComment(filter: CommentFilter): Comment!
```

Comment:
Missing rename

Reply:
the delete argument is used for querying, so it should be filter.

Reply:
It might be misscommunication. Requires chat